<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditorWindow</class>
 <widget class="QDialog" name="EditorWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>330</width>
    <height>324</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>330</width>
    <height>324</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>330</width>
    <height>324</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Edit record</string>
  </property>
  <widget class="QGroupBox" name="cfg_gBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>311</width>
     <height>251</height>
    </rect>
   </property>
   <property name="title">
    <string>Configuration</string>
   </property>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>31</y>
      <width>291</width>
      <height>211</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="order_id_hLayout">
       <item>
        <widget class="QLabel" name="id_label">
         <property name="text">
          <string>Order id:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="id_order_value_spinBox">
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>999999999</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="complete_date_hLayout">
       <item>
        <widget class="QLabel" name="complete_date_label">
         <property name="text">
          <string>Completion date:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDateEdit" name="complete_dateEdit">
         <property name="enabled">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="complete_order_name_hLayout">
       <item>
        <widget class="QLabel" name="completed_order_name_label">
         <property name="text">
          <string>Completed the order:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="complete_order_name_lEdit">
         <property name="enabled">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="status_label">
         <property name="text">
          <string>Status:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="status_comboBox"/>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>263</y>
     <width>311</width>
     <height>51</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QPushButton" name="confirm_btn">
    <property name="geometry">
     <rect>
      <x>114</x>
      <y>15</y>
      <width>80</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Confirm</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
